config/k8s/provider-deployment-runtime-config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+  apiVersion: pkg.crossplane.io/v1beta1
+  kind: DeploymentRuntimeConfig
+  metadata:
```

Comment:
We need the `argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true` annotation here too.

Reply:
Added the annotation.